lldb/source/Commands/CommandObjectDWIMPrint.cpp

```diff
@@ -173,7 +173,16 @@ bool CommandObjectDWIMPrint::DoExecute(StringRef command,
     auto *exe_scope = m_exe_ctx.GetBestExecutionContextScope();
     ValueObjectSP valobj_sp;
     ExpressionResults expr_result =
-        target.EvaluateExpression(expr, exe_scope, valobj_sp, eval_options);
+        target.EvaluateExpression(expr, exe_scope, valobj_sp, eval_options, &m_fixed_expression);
+    
+    // Only mention Fix-Its if the command applies them.
+    // The compiler errors can address any parsing issues after applying Fix-It(s).
```

Comment:
I'm not sure I understand what the second sentence is trying to convey.

Reply:
I did my best to reword/convert this comment from `CommandObjectExpression.cpp`:

```
  // We only tell you about the FixIt if we applied it.  The compiler errors
  // will suggest the FixIt if it parsed.
```

Reply:
Ah! The original comment makes more sense: I *think* what it's trying to say is:

LLDB only displays fixits if the expression evaluator applied them. Any compiler diagnostics LLDB displays refer to the modified expression with the fixit applied.

Reply:
Fixed to past-test because it comes printing the result.

```cpp
// Only mention Fix-Its if the expression evaluator applied them.
// Compiler errors refer to the final expression after applying Fix-It(s).
```